bindings/python/iota_sdk/client/_utils.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+class ClientUtils():
+
+    def hex_to_bech32(self, hex, bech32_hrp):
+        """Transforms a hex encoded address to a bech32 encoded address.
+        """
+        return self._call_method('hexToBech32', {
```

Comment:
Out of curiosity, `ClientUtils` doesn't extend something else so what is self?

Reply:
Oh is that done by `class Client(NodeCoreAPI, NodeIndexerAPI, HighLevelAPI, ClientUtils):` ?

Reply:
But how does it know which `_call_method` to use ?